src/components/LinkButton/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Button } from '@material-ui/core/';
+
+const LinkButton = React.forwardRef((props, ref) => {
+  const {
+    href = '/',
+    variant = 'outlined',
+    size = 'large',
+    color = 'default',
+    className = null,
+  } = props;
+
+  return (
+    <Button
+      ref={ref}
+      className={className}
+      variant={variant}
+      color={color}
+      size={size}
+      onClick={() => {
+        window.location.href = href;
```

Comment:
is it important to use `window.location.href = href;` ? can we avoid using it ?

This behaviour reloads the page. Our main goal is to write a SPA (single page application) which doesn't require to refresh a page.

Could you play around with this function? Maybe removing onClick will solve this problem.

Or. You can try to style a Material UI button. Instead of using <button />, you can try to use <a />.

Please, check the [docs](https://material-ui.com/api/button/).
![image](https://user-images.githubusercontent.com/2904139/92882427-98501900-f418-11ea-8bf6-3709b98ecc2a.png)

Reply:
Update in next commit